app/models/manageiq/providers/azure/inventory/collector/target_collection.rb

```diff
@@ -521,7 +521,7 @@ def infer_related_vm_ems_refs_db!
   def infer_related_vm_ems_refs_api!
     instances.each do |instance|
       # TODO(lsmola) add API scanning
-      target.add_target(:association => :flavors, :manager_ref => {:name => instance.properties.hardware_profile.vm_size.downcase})
```

Comment:
again, if ems_ref is downcased, we should keep this

Reply:
@Ladas Ok, restored the downcase, but switched it to use `ems_ref` instead of `name`.